### DISCUSSION TITLE
Valori entità rilevati oltre l'ultima settimana ...

### DISCUSSION THREAD

**Discussion by RobyBS** [indomus/forum#1795] [HUB: Home Assistant]
scusate la domanda sciocca .. 

ho un sensore in corridoio che misura la temperatura istantanea del locale "sensor.corridoio_temperature" .
Dallo storico vorrei oggi, 27 luglio 2022,  vedere i dati rilevati da 1 luglio a 15 luglio 2022 , ma mi dice che non ci sono dati da mostrare selezionando l'entità chiamata sopra ... 
è possibile ? 
mi fa vedere solo fina all'ultima settimana .. 
non è possibile recuperare i dati precedenti ? 

grazie

**Comment by 50494554524F** [comment]
home assistant memorizza i dati solo degli ultimi 10 giorni... se vuoi salvare più dati devi usare l'integrazione "recorder"

**Reply by RobyBS** [reply]
grazie 1000 ! .. 
provo a sbirciare e capire .. 

recuperare il regresso è impossibile ? senza tale integrazione ...
